packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/profil/translations/Messages_fr_FR.json

```diff
@@ -1,10 +1,12 @@
 {
-  "xdsl_access_profile_tooltip_interleaved": "<b>Interleaved :</b> Profil auto par défaut",
-  "xdsl_access_profile_tooltip_fast": "<b>Fast : </b> Profil auto en fastPath (ping amélioré sans correction d'erreur)",
-  "xdsl_access_profile_tooltip_ginp": "<b>G.INP : </b> Cette extension améliore la correction d’erreur sans affecter la performance de la ligne. Le DSLAM va utiliser le mode de protection (INP, Impulse Noise Protection) contre le bruit.",
-  "xdsl_access_profile_tooltip_auto": "<b>AUTO-[valeur]M :</b> Le DSLAM va synchroniser au maximum sur le débit",
-  "xdsl_access_profile_tooltip_snr": "<b>SNR-[valeur] : </b> Le DSLAM va analyser la qualité de la ligne et va negocier un rapport signal/bruit (SNR) au chiffre indique. Plus le chiffre est grand plus la stablilité de la ligne sera bonne mais plus le débit sera petit. Sur les profils sans SNR forces, le DSLAM essaie ;d'avoir un SNR de 6.",
   "xdsl_access_profile_tooltip_time": "La modification sera effective sous 1 à 5 minutes.",
   "xdsl_access_dslam_an_error_ocurred": "[DSLAM] Une erreur est survenue !",
-  "xdsl_access_profile_doing": "Changement de profil en cours"
-}
\ No newline at end of file
+  "xdsl_access_profile_doing": "Changement de profil en cours",
+  "xdsl_access_profile_tooltip_adsl_24m": "<b>ADSL_24M :</b> Débit maximum à 24000 / 1024 marge au bruit à 6 dB",
```

Comment:
I guess every `<b>...</b>` could have be turn into a `translate-values`.
IMO it sounds weird to translate these.

Maybe it could be enhance later.